lib/pymedphys/docs/tutes/index.rst

```diff
@@ -1,23 +0,0 @@
-Tutorials Overview
-==================
-
-At this current point in time there are no tutorials. 😭
-
-However, there is a community over at <https://pymedphys.discourse.group/> to
-help you find your feet using PyMedPhys. Please reach out over there and we'd
-love to help.
-
-The goal for these tutorials is to have a suite of Jupyter Notebooks that can
-be interactively run within the documentation to help Medical Physicists who
-have no Python experience begin to have success using the tools within the
-Medical Physics ecosystem. To build a suite of tutorials like this is a big
-task, and most of the developers of PyMedPhys are focused on developing
-PyMedPhys itself. Are you reading this and thinking that you might be able to
-manage and facilitate the creation of a suite of Notebooks like this? You
-certainly don't have to create them all by yourself, instead, just take the
-task by the horns, design the overall tutorial flow/course, and find people who
-can build each component.
```

Comment:
Maybe we could have a "I'm not a programmer but I'd still like to help out in some way, what can I do to help?" section? And then some variant of this paragraph can go there?

Reply:
I'm sorry I missed this! Will reflect somewhere suitable :)